chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -379,7 +369,7 @@ public void pack() {
     if(usePacked) {
       if(implementation instanceof NodeBasedOctree) {
         try {
-          implementation = new PackedOctree(implementation.getDepth(), ((NodeBasedOctree) implementation).root);
+          implementation = new BigPackedOctree(implementation.getDepth(), ((NodeBasedOctree) implementation).root);
```

Comment:
Should the selected packed octree implementation depend on the octree size (= node count)?

Reply:
This function no longer exist (even before that it wasn't used beyond my tests)